docs/examples/qaoa_qiskit.ipynb

```diff
@@ -0,0 +1,713 @@
+       "q_0: ┤ U2(0,π) ├┤ R(2*$\\beta$,0) ├──■──────────────────────■───░───■──»\n",
+       "     ├─────────┤├────────────────┤┌─┴─┐┌────────────────┐┌─┴─┐ ░   │  »\n",
+       "q_1: ┤ U2(0,π) ├┤ R(2*$\\beta$,0) ├┤ X ├┤ RZ(2*$\\gamma$) ├┤ X ├─░───┼──»\n",
+       "     ├─────────┤├────────────────┤└───┘└────────────────┘└───┘ ░   │  »\n",
+       "q_2: ┤ U2(0,π) ├┤ R(2*$\\beta$,0) ├─────────────────────────────░───┼──»\n",
+       "     ├─────────┤├────────────────┤                             ░ ┌─┴─┐»\n",
+       "q_3: ┤ U2(0,π) ├┤ R(2*$\\beta$,0) ├─────────────────────────────░─┤ X ├»\n",
+       "     └─────────┘└────────────────┘                             ░ └───┘»\n",
+       "«                             ░                              ░      »\n",
+       "«q_0: ────────────────────■───░──────────────────────────────░──────»\n",
+       "«                         │   ░                              ░      »\n",
+       "«q_1: ────────────────────┼───░───■──────────────────────■───░──────»\n",
+       "«                         │   ░ ┌─┴─┐┌────────────────┐┌─┴─┐ ░      »\n",
+       "«q_2: ────────────────────┼───░─┤ X ├┤ RZ(2*$\\gamma$) ├┤ X ├─░───■──»\n",
+       "«     ┌────────────────┐┌─┴─┐ ░ └───┘└────────────────┘└───┘ ░ ┌─┴─┐»\n",
+       "«q_3: ┤ RZ(2*$\\gamma$) ├┤ X ├─░──────────────────────────────░─┤ X ├»\n",
+       "«     └────────────────┘└───┘ ░                              ░ └───┘»\n",
+       "«                             ░ \n",
+       "«q_0: ────────────────────────░─\n",
+       "«                             ░ \n",
+       "«q_1: ────────────────────────░─\n",
+       "«                             ░ \n",
+       "«q_2: ────────────────────■───░─\n",
+       "«     ┌────────────────┐┌─┴─┐ ░ \n",
+       "«q_3: ┤ RZ(2*$\\gamma$) ├┤ X ├─░─\n",
+       "«     └────────────────┘└───┘ ░ "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc_qaoa = QuantumCircuit(nqubits)\n",
+    "\n",
+    "qc_qaoa.append(qc_0, [i for i in range(0, nqubits)])\n",
+    "qc_qaoa.append(qc_mix, [i for i in range(0, nqubits)])\n",
+    "qc_qaoa.append(qc_p, [i for i in range(0, nqubits)])\n",
+    "\n",
+    "qc_qaoa.decompose().decompose().draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next step is to find the optimal parameters $(\\boldsymbol{\\beta_{opt}}, \\boldsymbol{\\gamma_{opt}})$ such that the expectation value\n",
+    "\n",
+    "$$ \\langle \\psi(\\boldsymbol{\\beta}_{opt}, \\boldsymbol{\\gamma}_{opt}) \\rvert H_P \\lvert \n",
+    "\\psi(\\boldsymbol{\\beta}_{opt}, \\boldsymbol{\\gamma}_{opt}) \\rangle $$\n",
+    "\n",
+    "is minimized. Such an expectation can be obtained by doing measurement in the Z-basis. We use a classical optimization algorithm to find the optimal parameters. Following steps are involved as shown in the schematic \n",
```

Comment:
This break (`\n`) doesn't seem to be working in our version, instead the schematic pushes the last line down which looks funny.

Reply:
Looking into it